### ISSUE TITLE
Dante's inferno small screen flicker 

### ISSUE THREAD

**Issue by Levan7** [hrydgard/ppsspp#10006]
I noticed while playing the game that there is a small screen flicker that can be scene in this video  

I am using v1.4.2-473-gbb1f65118 build

![1](https://user-images.githubusercontent.com/5770076/31316248-f319aac8-ac3a-11e7-806e-d83a3eaac993.gif)
[video.zip](https://github.com/hrydgard/ppsspp/files/1366813/video.zip)

**Comment by unknownbrackets** [comment]
Does changing hardware transform affect this?  Does it happen in all backends - OpenGL, Vulkan, Direct3D 11?

Guessing it might be iether a mipmapping or z issue, but could be a lot of things.  Would probably help to have a GE recording - you can generate this on Windows by going to Debug -> GE Debugger, while this is happening, and then clicking "Record" in the top right.

It'll place a file in memstick/PSP/SYSTEM/DUMP/.  This is basically a recording of that one frame (sorta like a screenshot, but it tells us HOW the game drew it.)

To capture it while it's happening, you can use "Step Frame" while swinging around the weapon, probably.

-[Unknown]

**Comment by Sonnebirke** [comment]
First, sorry for by bad english.
I have the same problem as the thread creator. I may have the file that you requested. I'm not sure if I did everything right with this.
In addition, again my 3 pictures of the problem.
![dante 1](https://user-images.githubusercontent.com/37542915/37602482-b8ecb1d4-2b8c-11e8-84b1-3a629193c635.jpg)
![dante 2](https://user-images.githubusercontent.com/37542915/37602484-b9107e84-2b8c-11e8-94af-89d77b1c3da0.jpg)
![dante 3](https://user-images.githubusercontent.com/37542915/37602485-b93a093e-2b8c-11e8-9fe0-a2e911611cab.jpg)

**Comment by unknownbrackets** [comment]
Hmm, something suspicious is happening when it creates the bloom buffer.  Maybe this is somehow an estimate drawing size bug?

-[Unknown]

**Comment by Sonnebirke** [comment]
No, the problems occur immediately after the game starts. I played a little further, but it only got more worser.
![1](https://user-images.githubusercontent.com/37542915/37666701-7cf0f5e4-2c60-11e8-8d87-be7f92faad8d.png)
[ULES01387_0001.zip](https://github.com/hrydgard/ppsspp/files/1830135/ULES01387_0001.zip)
I also tried the settings to change, but no improvement. My system consists of an I5 760 and an AMD 7850 (2GB). Maybe a driver problem (February / 2018)?
This is a video of another person who has the same problem from 2016.
https://www.youtube.com/watch?v=D7QfsJC0uW0
